Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -91,6 +91,7 @@ The release process starts when the first build is provided to QA and ends when
 
 | Version | Release Engineer(s)  | QA effort   | Engineering effort          | Total effort  | Cut-off date  | Release date  |
 |---------|----------------------|-------------|-----------------------------|---------------|---------------|---------------|
+| 4.0.0  | Anders Ha <br> Ilya Puchka | Automated: `8h 21m`<br>Manual: `54h 00min`| `AV-519: ` | Total: **00h** | 10.06.2019 | 18.06.2019 |
```

Comment:
Total: 00h?

Reply:
Work in progress, I need to get engineering effort